lib/updatehub/updatehub.c

```diff
@@ -137,7 +205,6 @@ static bool
 is_compatible_hardware(struct resp_probe_some_boards *metadata_some_boards)
 {
 	int i;
-
```

Comment:
unrelated change, please remove. Having empty line here is good, so lets keep it.

Reply:
space added as suggested.